src/tox_lsr/test_scripts/runcollection.sh

```diff
@@ -12,19 +13,24 @@ SCRIPTDIR=$(readlink -f "$(dirname "$0")")
 # Collection commands that are run when `tox -e collection`:
 role=$(basename "${TOPDIR}")
 STABLE_TAG=${2:-master}
-LSR_ROLE2COLL_NAMESPACE="${LSR_ROLE2COLL_NAMESPACE:-fedora}"
-LSR_ROLE2COLL_NAME="${LSR_ROLE2COLL_NAME:-linux_system_roles}"
-cd "$LSR_TOX_ENV_DIR"
+export LSR_ROLE2COLL_NAMESPACE="${LSR_ROLE2COLL_NAMESPACE:-fedora}"
+export LSR_ROLE2COLL_NAME="${LSR_ROLE2COLL_NAME:-linux_system_roles}"
+
+# Since .tox is in .gitignore, ansible-test is skipped
+# if the collection path is in LSR_TOX_ENV_DIR.
+# Using a temp dir outside of .tox.
+export MY_LSR_TOX_ENV_DIR=$( mktemp -d -t tox-XXXXXXXX )
```

Comment:
you'll need to remove this directory upon exit - use something like `trap  "rm -rf $MY_LSR_TOX_ENV_DIR" 0`